storage_controller/src/service/safekeeper_service.rs

```diff
@@ -590,6 +591,20 @@ impl Service {
                 }
             }
             locked.safekeepers = Arc::new(safekeepers);
+            metrics::METRICS_REGISTRY
+                .metrics_group
+                .storage_controller_safekeeper_nodes
+                .set(locked.safekeepers.len() as i64);
+            metrics::METRICS_REGISTRY
+                .metrics_group
+                .storage_controller_https_safekeeper_nodes
+                .set(
+                    locked
+                        .safekeepers
+                        .values()
+                        .filter(|s| s.has_https_port())
+                        .count() as i64,
+                );
```

Comment:
I guess there is no delete_safekeeper() (yet?).

Reply:
cc @arpad-m, you'll have to think about this when adding an endpoint to delete safekeepers (akin to pageserver `fn node_drop`)

Reply:
> I guess there is no delete_safekeeper() (yet?).

Yes, only upsert